src/serialization/portable/PortableSerializer.ts

```diff
@@ -73,7 +69,7 @@ export class PortableSerializer implements Serializer {
                 input.position(backupPos);
             }
         }
-        var reader: PortableReader;
+        var reader: DefaultPortableReader;
```

Comment:
this change is unnecessary. `PortableReader` is the interface. Any `reader` that is `PortableReader` is suitable here.